src/typing/typeloadCacheHook.ml

```diff
@@ -10,6 +10,7 @@ type find_module_result =
 	| BinaryModule of HxbData.module_cache
 	| NoModule
 
+let find_module_extra_hook : (Common.context -> path -> module_def_extra) ref = ref (fun _ _ -> raise Not_found)
```

Comment:
Note: this is only here temporarily, it should be moved to a better suited place